Cargo.toml

```diff
@@ -34,10 +34,17 @@ quickcheck = "1"
 rand = "0.8.4"
 tide = "0.16"
 actix-web = "4"
+tokio = { version = "1", features = ["rt-multi-thread", "net", "macros", "signal"] }
+hyper = { version = "0.14.16", features = ["server", "http1", "tcp"] }
+lazy_static = "1.4"
 
 [build-dependencies]
 prost-build = { version = "0.9.0", optional = true }
 
+[[example]]
+name = "hyper"
+crate-type = ["bin"]
+
```

Comment:
Sorry for the potentialy stupid question: Why is this entry needed? I can already do `cargo run --example tide` today. Wouldn't `cargo run --example hyper` work just fine without the lines above?

Reply:
You're right, it's not needed, I mistakenly thought it was.